elife/config/etc-pgbouncer-pgbouncer.ini

```diff
@@ -0,0 +1,13 @@
+# https://pgbouncer.github.io/config.html
+
+[databases]
+{{ db_name }} = host={{ host }} dbname={{ db_name }} user={{ app_user }} password={{ app_pass }}
+
+[pgbouncer]
```

Comment:
is there a maximum number of connections to configure? We haven't had any problem due to overprovisioning resources though.

Reply:
there are a heap of options for fine tuning: https://pgbouncer.github.io/config.html

I think we'll be fine with the defaults, but I haven't actually tested this yet. It's just one of those things that has been nagging me for a while - that new-connection-per-query that Django does - and it seems like an easy win when combined with the psql-appdb work.